kvmagent/kvmagent/test/test_parsing_multipath.py

```
@@ -26,35 +35,35 @@ def test_write_conf():
     assert cmp(text3, text4) == 0
 
-    assert multipath.write_multipath_conf('multipath2.conf')
-    assert not multipath.write_multipath_conf('multipath2.conf')
+    assert write_multipath_conf('multipath2.conf')
+    assert not write_multipath_conf('multipath2.conf')
     with open("multipath2.conf",
          "r") as fd:
         text5 = multipath.parse_multipath_conf(fd)
     with open("multipath2.conf.out", "r") as fd1:
         text6 = multipath.parse_multipath_conf(fd1)
     assert cmp(text5, text6) == 0
```

Ruff (0.12.2)

Undefined name `cmp`

(F821)